pyproject.toml

```diff
@@ -136,7 +138,7 @@ good-names = [
 
 [tool.pytest.ini_options]
 minversion = "7.0"
-addopts = "-rs --cov=tripper --cov-report=term --doctest-modules"
+addopts = "-rs --cov=tripper --cov-report=term --doctest-modules --ignore=examples --ignore=tripper"
```

Comment:
```suggestion
addopts = "-rs --cov=tripper --cov-report=term --doctest-modules --ignore=examples"
```
I also do not like ignoring of examples, but tripper which is our code-base should definitely not be ignored.

Reply:
Replaced it with new `--doctest-ignore-import-errors`

Reply:
> Replaced it with new `--doctest-ignore-import-errors`

Reply:
> I also do not like ignoring of examples, but tripper which is our code-base should definitely not be ignored.

Replaced it with new `--doctest-ignore-import-errors`